File: Exercicios-LPA-main/Aula 15/ex9.py
import math

def circCirculo(raio: int) -> float:
    """
    Calcula a circunferência de um círculo dado seu raio.
    
    Args:
        raio (int): Raio do círculo
        
    Returns:
        float: Circunferência do círculo
    """
    return 2 * math.pi * raio

print(f"Circunferência com raio 5: {circCirculo(5):.2f}")
print(f"Circunferência com raio 10: {circCirculo(10):.2f}")
print(f"Circunferência com raio 15: {circCirculo(15):.2f}")

File: Exercicios-LPA-main/Aula 15/ex9.c
#include <stdio.h>
#include <math.h>

float circCirculo(int raio) {
    return 2 * 3.14159 * raio;
}

int main() {
    printf("Circunferência com raio 5: %.2f\n", circCirculo(5));
    printf("Circunferência com raio 10: %.2f\n", circCirculo(10));
    printf("Circunferência com raio 15: %.2f\n", circCirculo(15));
    return 0;
}